core/block/import/importer.go

```diff
@@ -48,13 +48,13 @@ const CName = "importer"
 const workerPoolSize = 10
 
 type Import struct {
-	converters      map[string]converter.Converter
-	s               *block.Service
-	oc              Creator
-	objectIDGetter  IDGetter
-	tempDirProvider core.TempDirProvider
-	sbtProvider     typeprovider.SmartBlockTypeProvider
-	fileSync        filesync.FileSync
+	converters       map[string]converter.Converter
+	s                *block.Service
+	oc               Creator
+	idGetterProvider objectid.IdGetterProvider
```

Comment:
maybe it's better to abstract getting id? 

```
type IdGetter interface {
	GetID(spaceId string, sn *converter.Snapshot, createdTime time.Time, getExisting bool) (string, treestorage.TreeStorageCreatePayload, error)
}
```

there is snapshot that contains smartBlockType, so we can directly call GetID, which will resolve required id provider automatically